#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Add leap second for 2005

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p0 < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p0 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1
esac
exit 0

--- timezone/leapseconds~	2005-04-05 21:20:40.000000000 -0400
+++ timezone/leapseconds	2005-08-06 02:03:20.000000000 -0400
@@ -1,4 +1,4 @@
-# @(#)leapseconds	7.19
+# @(#)leapseconds	7.20
 
 # Allowance for leapseconds added to each timezone file.
 
@@ -43,8 +43,10 @@
 Leap	1995	Dec	31	23:59:60	+	S
 Leap	1997	Jun	30	23:59:60	+	S
 Leap	1998	Dec	31	23:59:60	+	S
+Leap	2005	Dec	31	23:59:60	+	S
 
 #	INTERNATIONAL EARTH ROTATION AND REFERENCE SYSTEMS SERVICE (IERS)
+#
 # SERVICE INTERNATIONAL DE LA ROTATION TERRESTRE ET DES SYSTEMES DE REFERENCE
 #
 # SERVICE DE LA ROTATION TERRESTRE
@@ -52,29 +54,39 @@
 # 61, Av. de l'Observatoire 75014 PARIS (France)
 # Tel.      : 33 (0) 1 40 51 22 26
 # FAX       : 33 (0) 1 40 51 22 91
-# Internet  : services.iers@obspm.fr
-#
-#						Paris, 21 July 2004
-#
+# ...
+# http://hpiers.obspm.fr/eop-pc
 #
-#						Bulletin C 28
+#						Paris, 4 July 2005
+#							
+#						Bulletin C 30
 #
 #						To authorities responsible
 #						for the measurement and
 #						distribution of time
 #
-#			INFORMATION ON UTC - TAI
 #
-# NO positive leap second will be introduced at the end of December 2004.
+#                                    UTC TIME STEP
+#                             on the 1st of January 2006
+#
+# A positive leap second will be introduced at the end of December 2005.
+# The sequence of dates of the UTC second markers will be:
+#
+#			2005 December 31,     23h 59m 59s
+#			2005 December 31,     23h 59m 60s
+#			2006 January   1,      0h  0m  0s
+#
 # The difference between UTC and the International Atomic Time TAI is:
 #
-#	from 1999 January 1, 0h UTC, until further notice : UTC-TAI = -32 s
+# from 1999 January 1, 0h UTC, to 2006 January 1  0h UTC  : UTC-TAI = - 32s
+# from 2006 January 1, 0h UTC, until further notice       : UTC-TAI = - 33s
 #
-# Leap seconds can be introduced in UTC at the end of the months of December
+# Leap seconds can be introduced in UtC at the end of the months of December
 # or June, depending on the evolution of UT1-TAI. Bulletin C is mailed every
-# six months, either to announce a time step in UTC, or to confirm that there
+# six months, either to announce a time step in UTC or to confirm that there
 # will be no time step at the next possible date.
 #
 #					Daniel GAMBIS
-#					Director
+#					Head
 #					Earth Orientation Center of IERS
+#					Observatoire de Paris, France
